openrtb_ext/bidders.go

```diff
@@ -80,6 +80,7 @@ var coreBidderNames []BidderName = []BidderName{
 	BidderCadentApertureMX,
 	BidderCcx,
 	BidderCoinzilla,
+	BidderCointraffic,
```

Comment:
Super nitpick: please move this above `BidderCoinzilla` to maintain alphabetical order.